src/markdown-pages/install-with-kurl/load-balancers.md

```diff
@@ -0,0 +1,97 @@
+    3. Leave the address type as `IPv4`
+    4. Select the VPC and subnets where the kURL EC2 instances live under the `Network mapping` section
+    5. Add a Listener for 6443
+        1. Set the Procotol to `TCP`
+        2. Set the port to `6443`
+        3. Set the default action to forward to your `6443` target group that includes your kURL instance IPs
+
+**NOTE**: AWS Network Load Balancers do not have security groups associated with them. Ingress access to the IP addresses in your target group are defined by the security groups on the instances themselves. You need to ensure that the security group on your kURL EC2 instances allows traffic from the IP address of the AWS NLB. If the load balancer is public, you need to include the IP addresses of clients as well.
+
+## Azure Load Balancer (Internet-Facing)
+Internal Load Balancers in Azure do not support hairpinning. There are workarounds such as adding a second NIC with static routes or creating a loopback interface, but those are out of scope of this topic.
+
+1. Create an Azure Load Balancer
+    1. Set a name for the load balancer
+    2. Set the region to where the kURL EC2 instances live
+    3. Set the Type to `Public`
+    4. Set the SKU to `Standard`
+    5. Set the Tier to `Regional`
+    6. Create a new Public IP address for the load balancer or use an existing one
+2. Create a Backend Pool
+    1. Set a name for the pool
+    2. Set Backend Pool Configuration to `NIC`
+    3. Set IP Version to `IPv4`
+    4. Add the first kURL VM to the backend pool
+3. Create a Health Probe
+    1. Set a name for the probe
+    2. Set the protocol to `TCP`
+    3. Set the port to `6443`
+    4. If needed per your organization's requirements, change the defaults for `Interval` and `Unhealthy Threshold`
+4. Create a Load Balancer Rule
+    1. Set a name for the rule
+    2. Set IP Version to `IPv4`
+    3. Select the frontend IP address for the kURL load balancer
+    4. Set the protocol to `TCP`
+    5. Set the port to `6443`
+    6. Set the backend port to `6443`
+    7. Set the backend pool to the kURL pool you created earlier
+    8. Set `Outbound source network address translation` to `Outbound and inbound use the same IP`
+
+**NOTE**: Ingress access for Azure Load Balancers is defined by the inbound port rules on the VMs in the backend pool. You need to ensure that your port rules are configured to allow traffic for destination port `6443` in your VM's networking settings.
+
+## Google Cloud Platform (Internet-Facing)
+Hairpinning is supported by default when using a machine image provided by GCP. VM instances are automatically configured to route traffic destined for the load balancer to the loopback address of the VM where the traffic originated.
+
+**NOTE**: The below configuration does not cover the usage of instance groups.
+
+1. Create a Health Check
+    1. Set a name for the health check
+    2. Set the scope
```

Comment:
Is this `region` vs `global`? Do both work?

Reply:
Correct. I only tested `global`, so assumed `region` works as well, but for peace of mind i'm just going to check both now.